plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2394,4 +2394,4 @@ datalist_combobox.not_supported=<html>{0}<small> - not supported
 help_loader.no_entry_learn_more=<html><body>No help entry is defined. Learn more about <a href="{0}">{1}</a> on MCreator''s wiki.
 help_loader.learn_about=<br>Learn more about <a href="{0}">{1}</a> on MCreator''s wiki.
 help_loader.no_help_entry=No help entry is defined for <tt>{0}</tt> yet.<br>
-help_loader.no_help_found=<html><body>No help was found for the given help context.
+help_loader.no_help_found=<html><body>No help was found for the given help context.
```

Comment:
Revert this change as it is not needed

Reply:
I didn’t know it was changed. xD

Reply:
I can't fix it. The problem is caused by GitHub.

Reply:
GitHub editor may alter line changes, but git itself sure won't. I suggest you revert file from git terminal or idea.